pkg/types/config.go

```diff
@@ -47,7 +47,7 @@ func (c SecretConfig) BuildURL(path string) (string, error) {
 // BuildSecretsPathURL constructs a URL which can be used to identify a secret's path
 // subPath is the location of the secrets in the secrets engine
 func (c SecretConfig) BuildSecretsPathURL(subPath string) (string, error) {
-	return c.BuildURL(c.Path + subPath)
+	return c.BuildURL(c.SecretName + subPath)
```

Comment:
This resolves the need to include the leading `/` in the secretName
```suggestion
	return c.BuildURL(fmt.Sprintf("%s/%s,c.StoreBasePath, secretName)
```